src/main/kotlin/tools/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -18,10 +18,8 @@ fun Class<*>.isKotlinClass(): Boolean = this.isAnnotationPresent(Metadata::class
  *  map object.
  * @property nullIsSameAsDefault     Default false.  Whether to treat null values as absent when deserializing, thereby
  *  using the default value provided in Kotlin.
- * @property singletonSupport        Default: DISABLED.  Mode for singleton handling.
+ * @property enabledSingletonSupport        Default: DISABLED.  Mode for singleton handling.
```

Comment:
I think default is either `true` or `false`, not enum, since we have:

```
val enabledSingletonSupport: Boolean = KotlinFeature.SingletonSupport.enabledByDefault,
```